util/events.js

```diff
@@ -115,7 +115,7 @@ module.exports = async function (args, context = epochtal) {
             ws.subscribe(ws.data.event);
             event.connect(ws.data.steamid);
           } catch {
-            throw new UtilError("ERR_HANDLER", args, context);
+            new UtilError("ERR_HANDLER", args, context);
           }
```

Comment:
here and in 3 other cases, why do you not throw anymore? Just instantiating iirc will just log it, but that seems so dirty

Reply:
because there's no try/catch anywhere after this, throwing here literally just exits the program. there's also no benefit to forwarding this to anything but a log. these are websocket events, like "open" or "close". you can't exactly forward the error to a closed websocket, and it'd be incredibly unclean to print an error about a missing handler to... what, the socket itself?

Reply:
yeahhh uhh fine